CSI_Greenville_HIST8510_Final_Project-AR.qmd

```diff
@@ -0,0 +1,390 @@
+  summarize(count = n())
+
+ggplot(decade.count, aes(DEATH.TYPE, count)) + theme(axis.text.x=element_text(angle=90,hjust=1)) + geom_col() + facet_wrap(~DECADE) + labs(title = "Greenville Death by Decade")
+```
+
+In addition to the mode of death, the inquest data gives insight into the people dying in Greenville. It's important to notice what information is not included, especially for black South Carolinians.
+
+```{r}
+age.sorted <- CSI.GVL.TIDY %>%
+  mutate(AGE = if_else(str_detect(DECEASED.DESCRIPTION, "child"), "child",
+                       if_else(str_detect(DECEASED.DESCRIPTION, "infant"), "child",
+                        if_else(str_detect(DECEASED.DESCRIPTION, "boy"), "child", if_else(str_detect(DECEASED.DESCRIPTION, "son"), "child", "adult")))))
+
+head(age.sorted)
+```
+
+```{r}
+#adds sorting by race to dataframe previously sorted by age
+all.sorted <- age.sorted %>%
+  mutate(RACE = if_else(str_detect(DECEASED.DESCRIPTION, "black"), "black",
+                  if_else(str_detect(DECEASED.DESCRIPTION, "negro"), "black",
+                  if_else(str_detect(DECEASED.DESCRIPTION, "negroe"), "black",                          if_else(str_detect(DECEASED.DESCRIPTION, "colored"), "black",
+                  if_else(str_detect(DECEASED.DESCRIPTION, "free"), "black",
+                  if_else(str_detect(DECEASED.DESCRIPTION, "slave"), "black",                             "white")))))))
+
+head(all.sorted)
+```
+
+In examining the coroners inquest from Greenville County it is clear that this practice of refusing to name or further identify black individuals trickled down from the highest governmental offices to the local levels. It is also clear that infants and children of undetermined race were often not named in the inquests.
+
+Comparing the overall trends of the Greenville dataset with those from Anderson, Horry and Chesterfield provide additional insights. I chose these in an attempt to understand what regional differences account for the variation in numbers of inquests? If, as Berry claims, you can judge the effectiveness of government by the health of its citizens, what forces account for the variances in the recorded deaths across South Carolina during the 19th century? Also, Anderson's dataset is very similar in size to Greenville, whereas Horry County is the smallest and Chesterfield is one of the largest so they provide a representative sample.
+
+**CSI:Anderson**
+
+Anderson County follows a very similar development pattern to Greenville after the Revolutionary War. Population expansion in the early 1800s caused the larger Pendleton District to be divided into Anderson and Pickens Districts. Like Greenville, Anderson is primarily agricultural with a healthy industrial economy due to the establishment of mills.
+
+```{r}
+#load Anderson County data
+CSI.ANDERSON <- read.csv("https://docs.google.com/spreadsheets/d/e/2PACX-1vSZ5lXF7WTLe71GUg6Z_h6HgtoeQJ6BX1JQ7H6IgkGvrXSqzxIh720ElLPeWmzoXytW2O9J7bgL82Kg/pub?output=csv") 
```

Comment:
So each county has their own csv? You should download all of these and merge them into a single file with a County column. (but in R, not excel)